src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -764,9 +767,8 @@ export default class GroupComparisonStore {
     }
 
     @computed get mRNATabGrey() {
-        // grey out if
-        return (this.activeStudyIds.isComplete && this.activeStudyIds.result.length > 1) // more than one active study
-            || (this.activeGroups.isComplete && this.activeGroups.result.length !== 2) // not two active groups
+        return (this.activeGroups.isComplete && this.activeGroups.result.length < 2) //less than two active groups
```

Comment:
lets change the name of this to mRNATabDisabled or something.  so it's presentation agnostic